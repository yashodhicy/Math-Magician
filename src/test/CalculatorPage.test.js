import { render } from '@testing-library/react';
import CalculatorPage from '../Calculator';

describe('Should render as expected', () => {
  it('should work as expected', () => {
    const result = render(<CalculatorPage />);
    expect(result).toMatchSnapshot();
  });
});
