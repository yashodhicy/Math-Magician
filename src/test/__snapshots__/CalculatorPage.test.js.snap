// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render as expected should work as expected 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="calc"
      >
        <h2>
          Lets do some math!
        </h2>
        <div
          id="calculator-container"
        >
          <section
            id="output-screen"
          >
            <p>
               
              <output>
                 
              </output>
            </p>
          </section>
          <section
            id="input-screen"
          >
            <div
              id="input-buttons"
            >
              <button
                class="input"
                id="AC"
                type="button"
              >
                AC
              </button>
              <button
                class="input"
                id="+/-"
                type="button"
              >
                +/-
              </button>
              <button
                class="input"
                id="%"
                type="button"
              >
                %
              </button>
              <button
                class="input"
                id="7"
                type="button"
              >
                7
              </button>
              <button
                class="input"
                id="8"
                type="button"
              >
                8
              </button>
              <button
                class="input"
                id="9"
                type="button"
              >
                9
              </button>
              <button
                class="input"
                id="4"
                type="button"
              >
                4
              </button>
              <button
                class="input"
                id="5"
                type="button"
              >
                5
              </button>
              <button
                class="input"
                id="6"
                type="button"
              >
                6
              </button>
              <button
                class="input"
                id="1"
                type="button"
              >
                1
              </button>
              <button
                class="input"
                id="2"
                type="button"
              >
                2
              </button>
              <button
                class="input"
                id="3"
                type="button"
              >
                3
              </button>
              <button
                class="input wide"
                id="0"
                type="button"
              >
                0
              </button>
              <button
                class="input"
                id="."
                type="button"
              >
                .
              </button>
            </div>
            <div
              id="operations"
            >
              <button
                class="operation"
                id="รท"
                type="button"
              >
                รท
              </button>
              <button
                class="operation"
                id="x"
                type="button"
              >
                x
              </button>
              <button
                class="operation"
                id="-"
                type="button"
              >
                -
              </button>
              <button
                class="operation"
                id="+"
                type="button"
              >
                +
              </button>
              <button
                class="operation"
                id="="
                type="button"
              >
                =
              </button>
            </div>
          </section>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="calc"
    >
      <h2>
        Lets do some math!
      </h2>
      <div
        id="calculator-container"
      >
        <section
          id="output-screen"
        >
          <p>
             
            <output>
               
            </output>
          </p>
        </section>
        <section
          id="input-screen"
        >
          <div
            id="input-buttons"
          >
            <button
              class="input"
              id="AC"
              type="button"
            >
              AC
            </button>
            <button
              class="input"
              id="+/-"
              type="button"
            >
              +/-
            </button>
            <button
              class="input"
              id="%"
              type="button"
            >
              %
            </button>
            <button
              class="input"
              id="7"
              type="button"
            >
              7
            </button>
            <button
              class="input"
              id="8"
              type="button"
            >
              8
            </button>
            <button
              class="input"
              id="9"
              type="button"
            >
              9
            </button>
            <button
              class="input"
              id="4"
              type="button"
            >
              4
            </button>
            <button
              class="input"
              id="5"
              type="button"
            >
              5
            </button>
            <button
              class="input"
              id="6"
              type="button"
            >
              6
            </button>
            <button
              class="input"
              id="1"
              type="button"
            >
              1
            </button>
            <button
              class="input"
              id="2"
              type="button"
            >
              2
            </button>
            <button
              class="input"
              id="3"
              type="button"
            >
              3
            </button>
            <button
              class="input wide"
              id="0"
              type="button"
            >
              0
            </button>
            <button
              class="input"
              id="."
              type="button"
            >
              .
            </button>
          </div>
          <div
            id="operations"
          >
            <button
              class="operation"
              id="รท"
              type="button"
            >
              รท
            </button>
            <button
              class="operation"
              id="x"
              type="button"
            >
              x
            </button>
            <button
              class="operation"
              id="-"
              type="button"
            >
              -
            </button>
            <button
              class="operation"
              id="+"
              type="button"
            >
              +
            </button>
            <button
              class="operation"
              id="="
              type="button"
            >
              =
            </button>
          </div>
        </section>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
